kratos/tests/cpp_tests/utilities/test_delaunator_utilities.cpp

```diff
@@ -0,0 +1,41 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//
+//
+
+// Project includes
+#include "testing/testing.h"
+#include "containers/model.h"
+#include "utilities/delaunator_utilities.h"
+
+namespace Kratos {
+namespace Testing {
+
+KRATOS_TEST_CASE_IN_SUITE(CreateTriangleMeshFromNodes, KratosCoreFastSuite)
+{
+    Model this_model;
+    ModelPart& r_model_part = this_model.CreateModelPart("Main", 2);
+
+    // First we create the nodes
+    r_model_part.CreateNewNode(1, 0.0 , 0.0 , 0.0);
+    r_model_part.CreateNewNode(2, 1.0 , 0.0 , 0.0);
+    r_model_part.CreateNewNode(3, 1.0 , 1.0 , 0.0);
+    r_model_part.CreateNewNode(4, 0.0 , 1.0 , 0.0);
+    r_model_part.CreateNewNode(5, 2.0 , 0.0 , 0.0);
+    r_model_part.CreateNewNode(6, 2.0 , 1.0 , 0.0);
+
+    DelaunatorUtilities::CreateTriangleMeshFromNodes(r_model_part);
+
+    KRATOS_CHECK(r_model_part.Elements().size() == 4);
```

Comment:
shouldn't you also check the connectivities?

Reply:
I did that when the utilities were using triangle and the delaunator library, but after removing delaunator I just check the total area (see the checks), that is the relevant check, because the connectivities may change depending of the Delaunay parameters considered, but at the end the total area is the correct thing to check (well, is the alpha is to small no triangles are going to be detected, but we will not consider this case)

Reply:
ok for me, but there is no check for Area, only number of elements

Reply:
OK, you mean in this test

Reply:
I can add an area to this test too

Reply:
I added a test based on area. Funny thing is that I added directly from GitHub and it worked (I am evil ;))